src/features/swap/index.tsx

```diff
@@ -0,0 +1,207 @@
+
+  const groupKey = 'swap-order-status'
+  const listeners = useMemo<CowEventListeners>(() => {
+    return [
+      {
+        event: CowEvents.ON_TOAST_MESSAGE,
+        handler: (event) => {
+          console.info('🍞 New toast message:', event)
+          const { message, messageType, data } = event
+          switch (messageType) {
+            // @ts-ignore
+            case 'ORDER_CREATED':
+              dispatch(
+                showNotification({
+                  title: 'Order created',
+                  message: 'Order created but waiting for presignature',
+                  groupKey,
+                  variant: 'info',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_PRESIGNED':
+              dispatch(
+                showNotification({
+                  title: 'Order presigned waiting for fullfillment',
+                  message: 'Order was presigned and waiting for match',
+                  groupKey,
+                  variant: 'info',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_FULFILLED':
+              dispatch(
+                showNotification({
+                  title: 'Order fullfilled',
+                  message: 'Order was fullfilled',
+                  groupKey,
+                  variant: 'info',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_EXPIRED':
+              dispatch(
+                showNotification({
+                  title: 'Order expired',
+                  message,
+                  groupKey,
+                  variant: 'warning',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_CANCELLED':
+              dispatch(
+                showNotification({
+                  title: 'Order cancelled',
+                  message,
+                  groupKey,
+                  variant: 'warning',
+                }),
+              )
+              break
+          }
+        },
+      },
+    ]
+  }, [dispatch])
+
+  const [params, setParams] = useState<CowSwapWidgetParams | null>(null)
+  useEffect(() => {
+    setParams({
+      appCode: 'Safe Wallet Swaps', // Name of your app (max 50 characters)
+      width: '100%', // Width in pixels (or 100% to use all available space)
+      height: '860px',
+      // provider: safeAppWeb3Provider, // Ethereum EIP-1193 provider. For a quick test, you can pass `window.ethereum`, but consider using something like https://web3modal.com
+      chainId, // 1 (Mainnet), 5 (Goerli), 100 (Gnosis)
+      // standaloneMode: false,
+      disableToastMessages: true,
+      disablePostedOrderConfirmationModal: true,
+      hideLogo: true,
+      hideNetworkSelector: true,
+      sounds: {
+        orderError: null,
+        orderExecuted: null,
+        postOrder: null,
```

Comment:
🥲 Why not adding something not too annoying (like a laser gun shoot)
The mooooo  sound is the most praised feedback. Every single week we get people comments on it on the feedback button

Reply:
sadly, marketing is against it for the moment.